### PR TITLE
Exclude older PHP versions from .travis.yml

### PR DESCRIPTION
**Pull request by LukasReschke**
cc @karlitschek @MorrisJobke @nickvergessen

### CODE REVIEW

**Comment by blizzz** [comment]
LGTM

**Comment by LukasReschke** [comment]
Awesome, Travis execution is also much faster.

Before: 20 minutes 50 secs
Now: 9 minutes 16 secs

🚀

**Comment by karlitschek** [comment]
👍

File: index.php [hunk 1]
```diff
@@ -25,10 +25,10 @@
...
  */
 
-// Show warning if a PHP version below 5.4.0 is used, this has to happen here
-// because base.php will already use 5.4 syntax.
-if (version_compare(PHP_VERSION, '5.4.0') === -1) {
-	echo 'This version of ownCloud requires at least PHP 5.4.0<br/>';
+// Show warning if a PHP version below 5.6.0 is used, this has to happen here
+// because base.php will already use 5.6 syntax.
+if (version_compare(PHP_VERSION, '5.6.0') === -1) {
+	echo 'This version of Nextcloud requires at least PHP 5.6.0<br/>';
```
Comment (nickvergessen) [inline index.php]: The name change is already backported?

Comment (LukasReschke) [inline index.php]: v9.0.51 fixed that: https://github.com/nextcloud/server/blob/v9.0.51/index.php#L30

=> https://github.com/nextcloud/server/commit/8abf6e0ed1c50c1ace8e83bd5940ae7156726ba4

**Pull request merged**